pkg/services/sqlstore/dashboard_snapshot_test.go

```diff
@@ -5,169 +5,169 @@ import (
+			}
+			err := SearchDashboardSnapshots(&query)
+			require.NoError(t, err)
 
-			Convey("And the user has the editor role and has not created any snapshot", func() {
-				Convey("Should not return any snapshots", func() {
-					query := models.GetDashboardSnapshotsQuery{
-						OrgId:        1,
-						SignedInUser: &models.SignedInUser{OrgRole: models.ROLE_EDITOR, UserId: 2},
-					}
-					err := SearchDashboardSnapshots(&query)
-					So(err, ShouldBeNil)
-
-					So(query.Result, ShouldNotBeNil)
-					So(len(query.Result), ShouldEqual, 0)
-				})
+			t.Run("Should not return any snapshots", func(t *testing.T) {
+				assert.NotNil(t, query.Result)
+				assert.Equal(t, len(query.Result), 0)
 			})
+		})
+
+		t.Run("And the user is anonymous", func(t *testing.T) {
+			cmd := models.CreateDashboardSnapshotCommand{
+				Key:       "strangesnapshotwithuserid0",
+				DeleteKey: "adeletekey",
+				Dashboard: simplejson.NewFromAny(map[string]interface{}{
+					"hello": "mupp",
+				}),
+				UserId: 0,
+				OrgId:  1,
+			}
+			err := CreateDashboardSnapshot(&cmd)
+			require.NoError(t, err)
 
-			Convey("And the user is anonymous", func() {
-				cmd := models.CreateDashboardSnapshotCommand{
-					Key:       "strangesnapshotwithuserid0",
-					DeleteKey: "adeletekey",
-					Dashboard: simplejson.NewFromAny(map[string]interface{}{
-						"hello": "mupp",
-					}),
-					UserId: 0,
-					OrgId:  1,
+			t.Run("Should not return any snapshots", func(t *testing.T) {
+				query := models.GetDashboardSnapshotsQuery{
+					OrgId:        1,
+					SignedInUser: &models.SignedInUser{OrgRole: models.ROLE_EDITOR, IsAnonymous: true, UserId: 0},
 				}
-				err := CreateDashboardSnapshot(&cmd)
-				So(err, ShouldBeNil)
-
-				Convey("Should not return any snapshots", func() {
-					query := models.GetDashboardSnapshotsQuery{
-						OrgId:        1,
-						SignedInUser: &models.SignedInUser{OrgRole: models.ROLE_EDITOR, IsAnonymous: true, UserId: 0},
-					}
-					err := SearchDashboardSnapshots(&query)
-					So(err, ShouldBeNil)
-
-					So(query.Result, ShouldNotBeNil)
-					So(len(query.Result), ShouldEqual, 0)
-				})
+				err := SearchDashboardSnapshots(&query)
+				require.NoError(t, err)
+
+				assert.NotNil(t, query.Result)
+				assert.Equal(t, len(query.Result), 0)
 			})
+		})
 
-			Convey("Should have encrypted dashboard data", func() {
-				original, err := cmd.Dashboard.Encode()
-				So(err, ShouldBeNil)
+		t.Run("Should have encrypted dashboard data", func(t *testing.T) {
+			original, err := cmd.Dashboard.Encode()
+			require.NoError(t, err)
 
-				decrypted, err := util.Decrypt(cmd.Result.DashboardEncrypted, setting.SecretKey)
-				So(err, ShouldBeNil)
+			decrypted, err := cmd.Result.DashboardEncrypted.DecodeAndDecrypt()
+			require.NoError(t, err)
 
-				So(bytes.Equal(decrypted, original), ShouldEqual, true)
-			})
+			assert.Equal(t, bytes.Equal(decrypted, original), true)
```

Comment:
Shouldn't this also work?
```suggestion
			assert.Equal(t, original, decrypted, original)
```